pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentDispatcherMultipleConsumers.java

```diff
@@ -85,6 +85,10 @@
  */
 public class PersistentDispatcherMultipleConsumers extends AbstractDispatcherMultipleConsumers
         implements Dispatcher, ReadEntriesCallback {
+    // rescheduling a read after no entries are dispatches will be delayed by this duration using a backoff
+    private static final int RESCHEDULE_READ_INITIAL_DELAY_MS = 100;
+    // maximum delay for rescheduling a read after no entries are dispatched
+    private static final int RESCHEDULE_READ_INITIAL_MAX_DELAY_MS = 5000;
```

Comment:
I think we should have these parameters configurable

Reply:
This is now addressed. I also made the max delay 1000 instead of 5000.